shopinvader_algolia_stock/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Akretion (http://www.akretion.com)
+# Copyright 2018 ACSONE SA/NV
+# Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Connector Search Engine Algolia stock",
+    "summary": "Shopinvader stock algolia",
+    'description': """
+        Connector search engine for Algolia Stock""",
```

Comment:
@acsonefho The description should be more meaningful.  'This module adds the stock levele of your products to the informations exported to Aalgolia'